frontend/src/lib/components/Errors/ErrorDisplay.tsx

```diff
@@ -121,19 +121,25 @@ function FrameContextLine({
     )
 }
 function ChainedStackTraces({ exceptionList }: { exceptionList: ErrorTrackingException[] }): JSX.Element {
-    const [showAllFrames, setShowAllFrames] = useState(false)
+    const hasAnyInApp = exceptionList
+        .map(({ stacktrace }) => stacktrace?.frames?.some((frame) => frame.in_app))
+        .some(Boolean)
```

Comment:
Would a double some work?
```suggestion
    const hasAnyInApp = exceptionList
        .some(({ stacktrace }) => stacktrace?.frames?.some(({ in_app }) => in_app)
    )
```

Reply:
Huh, apparently. Eventually I'll get a feel for this type system